ppl/src/main/antlr/OpenSearchPPLParser.g4

```diff
@@ -127,6 +128,27 @@ patternsMethod
    | REGEX
    ;
 
+fillnullCommand
```

Comment:
@normanj-bitquill I dont see FILLNULL in the `keywordsCanBeId ` list ?
https://github.com/opensearch-project/sql/blob/652b959b550f0d9ddd9f654c5dd674e159e2597e/ppl/src/main/antlr/OpenSearchPPLParser.g4#L843

Reply:
Updated. I see how `fillnull` is in the keywordsCanBeId list for Spark.